src/Bundle/CoreBundle/Model/StateSettings.php

```diff
@@ -0,0 +1,327 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: stefankamsker
+ * Date: 14.09.18
+ * Time: 15:16
+ */
+
+namespace UniteCMS\CoreBundle\Model;
+
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+use Symfony\Component\Validator\Validation;
+use UniteCMS\CoreBundle\Field\FieldableFieldSettings;
+use UniteCMS\CoreBundle\Model\StatePlace;
+use UniteCMS\CoreBundle\Model\StateTransition;
+
+/**
+ * We use this model only for validation!
+ */
+class StateSettings
+{
+
+    /**
+     * @var string
+     * @Assert\Type(type="string", message="workflow_invalid_initial_place")
+     * @Assert\NotBlank(message="workflow_invalid_initial_place")
+     * @Assert\Choice(callback="getPlacesIdentifiers", message="workflow_invalid_initial_place")
+     */
+    private $initialPlace;
+
+    /**
+     * @var StatePlace[]
+     * @Assert\Valid
+     * @Assert\NotBlank(message="workflow_invalid_places")

```

Comment:
If you want to tell the user, that a value is required, use "not_blank" or "not_null" message. At the moment the user would get a "he given places are invalid." message which seems not very clear to me

Reply:
You should also check all other messages and use existing validation messages if possible.